lib/api/controllers/documentController.js

```diff
@@ -46,11 +48,35 @@ function DocumentController (kuzzle) {
    */
   this.get = function documentGet (request) {
     assertHasId(request, 'document:get');
+    assertHasIndexAndCollection(request, 'document:get');
 
     return engine.get(request);
   };
 
   /**
+   * Get specific documents according to given ids
+   *
+   * @param {Request} request
+   * @returns {Promise<Object>}
+   */
+  this.mGet = function documentMGet (request) {
+    assertHasBody(request, 'document:mGet');
+    assertBodyHasAttribute(request, 'ids', 'document:mGet');
+    assertHasIndexAndCollection(request, 'document:mGet');
+
+    if (!Array.isArray(request.input.body.ids)) {
```

Comment:
notpicking: can't we put this lines into a new `assertBodyAttributeType(request, attribute, type)` method ?